pandas/core/groupby/groupby.py

```diff
@@ -2433,21 +2447,40 @@ def _get_cythonized_result(
             name = obj.name
             values = obj._values
 
+            if numeric_only and not is_numeric_dtype(values):
+                continue
+
             if aggregate:
                 result_sz = ngroups
             else:
                 result_sz = len(values)
 
-            result = np.zeros(result_sz, dtype=cython_dtype)
-            func = partial(base_func, result, labels)
+            if needs_at_least2d:
```

Comment:
can you use at_least2d here (the numpy function) this is what i meant.

Reply:
Ah - I see. I don't believe so; we'd like to convert [1, 2] to [[1], [2]]. atleast_2d will do output [[1, 2]] instead.